flintrock/flintrock.py

```diff
@@ -334,6 +339,22 @@ def launch(
             )
         services += [spark]
 
+    # Format ec2 tags
```

Comment:
Once it's split out into a function, we should probably also add a test or two to `tests/test_ec2.py`.